code/modules/mob/living/carbon/carbon.dm

```diff
@@ -443,7 +443,7 @@
 	return
 
 /mob/living/carbon/slip(var/slipped_on,stun_duration=8)
-	if(buckled)
+	if(buckled || istype(species, /datum/species/skrell))

```

Comment:
The safer, and more uniform way of doing this is: `if (buckled || get_species == "Skrell")`

Running `istype()` on `null`, in the case of simple mobs, might generate runtime errors. Whereas `get_species` is correctly protected and guarded for the entire `mob` object tree.